app/src/main/java/com/coveros/coverosmobileapp/blogpost/BlogPost.java

```diff
@@ -50,6 +50,23 @@
         this.content = "<h3>" + this.title + "</h3><h4>" + author + "</h4><h5>" + this.date + "</h5>" + StringEscapeUtils.unescapeHtml4(blogPostJson.get("content").getAsJsonObject().get("rendered").getAsString());
     }
 
+    // used for testing so creating a JsonObject is not required
+    public BlogPost(String id, String author, String date, String content, String title) {
```

Comment:
Is this really a constructor used only for testing? There should be a better way.